kivy/uix/textinput.py

```diff
@@ -1402,7 +1402,8 @@ def on_touch_up(self, touch):
             # show Bubble
             win = EventLoop.window
             if self._selection_to != self._selection_from:
-                self._show_cut_copy_paste(touch.pos, win)
+                touch_rel_pos = (touch.pos[0] - self.x, touch.pos[1] - self.y)
```

Comment:
There should be no adjustment made outside the cut copy paste function as positioning is done in it.

Doing it in multiple places will lead to confusion.